src/KeyboardInputModal.scss

```diff
@@ -1,9 +1,7 @@
 @import '_variables';
 
 .KeyboardInputModal__content {
-    overflow: auto;
     width: 45rem;
-    max-height: 30rem;
```

Comment:
I suggest setting max-height back on the content container, and set it to use dynamic height I suggested on C2LC-531.

```
max-height: calc(100vh - 15rem);
```

Reply:
I used the suggested approach.